src/components/Workspace/Workspace.tsx

```diff
@@ -63,10 +62,17 @@ export interface StatusData {
 export async function fetchProblemData(
   problemID: string
 ): Promise<ProblemData | null> {
-  const url = `${judgePrefix}/problem/${problemID}`;
-  const response = await fetch(url);
-  if (response.status !== 200) return null;
-  return await response.json();
+  const response = await fetch(
+    'https://raw.githubusercontent.com/cpinitiative/usaco-problems/main/problems.json'
+  );
+  const data = await response.json();
+  const problem = data[problemID];
```

Comment:
We should probably explicitly check to see if `problemID` exists inside data, and return an appropriate response if it doesn't? Otherwise it might not work on the client-side, or it might clutter our logs.